index-2.qmd

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
nit: Do you intend to link this page from `_quarto.yml` or from any other page? Also consider naming it something other than `index-2.qmd`.